src/frontends/tensorflow_lite/include/openvino/frontend/tensorflow_lite/node_context.hpp

```diff
@@ -12,6 +12,9 @@ namespace ov {
 namespace frontend {
 namespace tensorflow_lite {
 
+using SubGraphFuncs = std::vector<std::function<std::shared_ptr<ov::Model>()>>;
+using SubGraphFuncsPtr = std::shared_ptr<SubGraphFuncs>;
```

Comment:
Sorry maybe for stupid question, but why do you need to have a shared pointer on vector?
I think we can pass vector by reference...